application/exeptions.py

```diff
@@ -0,0 +1,4 @@
+class BaseAppException(Exception):
+    def __init__(self, message: str):
```

Comment:
Без конструктора
```suggestion
    message: str = ''
```

Тогда не придется каждый раз вызывая эту ошибку делать
`raise SomeException(message='Message text')`